src/Components/Main/Main.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { usePokemon } from '../../hooks/pokehook';
+import Controls from '../Controls/Controls';
+import PokeCard from '../PokeCard/PokeCard.js';
+import './Main.css';
+
+export default function Main() {
+  const { pokemon, types, loading, handleTypeChange } = usePokemon();
+  if (loading === true) {
```

Comment:
the argument is redundant.  you do not need to set it equal to true.  Also, try and refactor this into a ternary.